builtin.aqua

```diff
@@ -304,10 +295,15 @@ service Dist("dist"):
     -- Get the interface of a module
     get_module_interface(module_hash: string) -> Interface
 
-    -- Creates Blueprint structure from from blueprint name and dependencies (modules)
-    make_blueprint(name: string, dependencies: []Dependency) -> AddBlueprint
-    -- Add a blueprint to the node
-    add_blueprint(blueprint: AddBlueprint) -> string
+    -- Creates IPLD Blueprint structure from from blueprint name and dependencies (modules)
+    make_blueprint(name: string, dependencies: []CID) -> IPLDBlueprint
+    -- Add an IPLD blueprint to the node
+    add_blueprint(blueprint: IPLDBlueprint) -> string
+
+    -- Loads blueprint by copying it from Particle Vault directory
+    -- Arguments:
+    --  blueprint_path   – path or a filename
```

Comment:
I don't get it. Is it a path or a filename? What should it look like?

Reply:
This is a path in the vault, smth like `/tmp/vault/blueprint.json`. I will clarify this comment